packages/tracing/src/index.bundle.ts

```diff
@@ -0,0 +1,79 @@
+export {
+  Breadcrumb,
+  Request,
+  SdkInfo,
+  Event,
+  Exception,
+  Response,
+  Severity,
+  StackFrame,
+  Stacktrace,
+  Status,
+  Thread,
+  User,
+} from '@sentry/types';
+
+export {
+  addGlobalEventProcessor,
+  addBreadcrumb,
+  captureException,
+  captureEvent,
+  captureMessage,
+  configureScope,
+  getHubFromCarrier,
+  getCurrentHub,
+  Hub,
+  Scope,
+  setContext,
+  setExtra,
+  setExtras,
+  setTag,
+  setTags,
+  setUser,
+  Transports,
+  withScope,
+} from '@sentry/browser';
+
+export { BrowserOptions } from '@sentry/browser';
+export { BrowserClient, ReportDialogOptions } from '@sentry/browser';
+export {
+  defaultIntegrations,
+  forceLoad,
+  init,
+  lastEventId,
+  onLoad,
+  showReportDialog,
+  flush,
+  close,
+  wrap,
+} from '@sentry/browser';
+export { SDK_NAME, SDK_VERSION } from '@sentry/browser';
+
+import { Integrations as BrowserIntegrations } from '@sentry/browser';
+import { getGlobalObject } from '@sentry/utils';
+
+import { BrowserTracing } from './browser';
+import { addExtensionMethods } from './hubextensions';
+
+export { Span, TRACEPARENT_REGEXP } from './span';
+
+let windowIntegrations = {};
+
+// This block is needed to add compatibility with the integrations packages when used with a CDN
+// tslint:disable: no-unsafe-any
+const _window = getGlobalObject<Window>();
+if (_window.Sentry && _window.Sentry.Integrations) {
+  windowIntegrations = _window.Sentry.Integrations;
+}
+// tslint:enable: no-unsafe-any
+
+const INTEGRATIONS = {
+  ...windowIntegrations,
+  ...BrowserIntegrations,
+  ...BrowserTracing,
+};
+
+export { INTEGRATIONS as Integrations };
+
+// We are patching the global object with our hub extension methods
+addExtensionMethods();
```

Comment:
We still have to move this into an option, but I'll save that for another PR.

Something like
```
Sentry.init({
  extensions: [apm]
})
```

Reply:
Let's deal with it on a follow up PR